logging/cluster-logging-release-notes.adoc

```diff
@@ -8,235 +8,14 @@ toc::[]
-// [id="openshift-logging-5-0-asynchronous-errata-updates"]
-// == Asynchronous errata updates
-//
-// Security, bug fix, and enhancement updates for {ProductName} 5.0 are released as asynchronous errata through the Red Hat Network. All {ProductName} 5.0 errata are https://access.redhat.com/downloads/[available on the Red Hat Customer Portal]. See the https://access.redhat.com/support/policy/updates/openshift#logging[{ProductName} Life Cycle] for more information about asynchronous errata.
-// // TBD Update https://access.redhat.com/downloads/ to something like https://access.redhat.com/downloads/content/201/ once the Logging product has been released.
-//
-// Red Hat Customer Portal users can enable errata notifications in the account settings for Red Hat Subscription Management (RHSM). When errata notifications are enabled, users are notified via email whenever new errata relevant to their registered systems are released.
-//
-// [NOTE]
-// ====
-// Red Hat Customer Portal user accounts must have systems registered and consuming {ProductName} entitlements for {ProductName} errata notification emails to generate.
-// ====
-//
-// This section will continue to be updated over time to provide notes on enhancements and bug fixes for future asynchronous errata releases of {ProductName} 5.0. Versioned asynchronous releases, for example with the form {ProductName} 5.0.z, will be detailed in subsections. In addition, releases in which the errata text cannot fit in the space provided by the advisory will be detailed in subsections that follow.
-//
-// [IMPORTANT]
-// ====
-// For any {ProductName} release, always review the instructions on xref:../updating/updating-cluster.adoc#TBD[updating your cluster] properly.
-// ====
-//
-// [id="openshift-logging-5-0-0-ga"]
-// === RHBA-2020:66974-04 Errata Advisory for Openshift Logging 5.0.0
-//
-// (link:https://errata.devel.redhat.com/docs/show/66974[RHBA-2020:66974-04 Errata Advisory for Openshift Logging 5.0.0]) is now available. New features, changes,
-//
-// Issued: 2021-02-24
-//
-// {ProductName} release 5.0 is now available. The list of bug fixes that are included in the update is documented in the link:https://errata.devel.redhat.com/docs/show/66974[RHBA-2020:66974-04] advisory.
-
-// UNUSED IN THIS 5.0.0 RELEASE - KEEP THIS BOILERPLATE FOR FUTURE RELEASES
-// The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2020:5678[RHBA-2020:5678] advisory.
-//
-// Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
-//
-// link:https://access.redhat.com/solutions/<ARTICLE_ID>[{ProductName} 5.0.0 container image list]
+include::modules/cluster-logging-release-notes-5.0.0.adoc[leveloffset=+1]
+//include::modules/cluster-logging-release-notes-5.0.1.adoc[leveloffset=+1]
```

Comment:
@vikram-redhat  I've commented out the 5.0.1 RNs for now so we can merge the general changes now. Then I'll create another PR for the 5.0.1 changes.